### PR TITLE
Updated contributors to use istio 1.9 rbac

### PR DESCRIPTION
**Pull request by andres-de-castro**

### CODE REVIEW

File: content/en/docs/components/multi-tenancy/getting-started.md [hunk 1]
```diff
@@ -319,36 +316,30 @@ subjects:
...
-  namespace: profileName # replace with the namespace/profile name that you are adding contribbitors to
+  namespace: profileName # replace with the namespace/profile name that you are adding contributors to
 spec:
-  roleRef:
-    kind: ServiceRole
-    name: ns-access-istio
-  subjects:
-  - properties:
-      request.headers[kubeflow-userid]: accounts.google.com:userid@email.com   # replace with the email of the user from your Active Directory case sensitive
-      # for GCP, use x-goog-authenticated-user-email instead of kubeflow-userid for authentication purpose
```
Comment (Bobgy) [inline content/en/docs/components/multi-tenancy/getting-started.md:343]: Please keep this comment for platform differences

**Review by Bobgy — COMMENTED** [review]
Thank you for updating the doc!
Great work!
/lgtm

/cc @yanniszark for review too

Also you'll need to sign the CLA, before I can merge this.

**Review by Arhell — COMMENTED** [review]
about CLA
https://github.com/kubeflow/website/pull/2790#issuecomment-866198976

**Comment by andres-de-castro** [comment]
@googlebot I signed it!

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you for keeping the docs updated!
Really appreciate it : )

**Pull request merged**